media/inf/kscaptur.inf

```diff
@@ -387,11 +377,8 @@ Plugin_PropertyObject="Knotenschnittstellen-Aggregierungshandler"
 
 [Strings.0a]
 VfWWDMDriverDesc="Dispositivo de captura de vídeo por WDM"
-
 Filter_TVAudio="Audio de TV por WDM"
 Filter_TVTuner="Receptor de TV por WDM"
-
-;Node.Crossbar="Crossbar"
```

Comment:
Revert and uncomment?

Reply:
no it is 1:1 the english string. So why should we duplicate it?